common/src/main/java/org/apache/falcon/security/AuthenticationInitializationService.java

```diff
@@ -83,8 +84,13 @@ public void init() throws FalconException {
             String authTokenValidity = StartupProperties.get().getProperty(AUTH_TOKEN_VALIDITY_SECONDS);
             long validateFrequency;
             try {
+                // -100 so that revalidation is done before expiry.
                 validateFrequency = (StringUtils.isNotEmpty(authTokenValidity))
-                        ? Long.parseLong(authTokenValidity) : 86400;
+                        ? (Long.parseLong(authTokenValidity) - 100) : DEFAULT_VALIDATE_FREQUENCY_SECS;
+                if (validateFrequency < 0) {
+                    throw new NumberFormatException("Value provided for startup property \""
+                            + AUTH_TOKEN_VALIDITY_SECONDS + "\" is too small.");
```

Comment:
Can the logger statement be changed to saying should be greater than 100, which will help user rather than trial and attempt